packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -327,10 +327,18 @@ def expected_service_running(
     with log_context(logging.INFO, msg="Waiting for node to run") as ctx:
         waiter = SocketIONodeProgressCompleteWaiter(node_id=node_id, logger=ctx.logger)
         service_running = ServiceRunning(iframe_locator=None)
-        with websocket.expect_event("framereceived", waiter, timeout=timeout):
-            if press_start_button:
-                _trigger_service_start(page, node_id)
-            yield service_running
+
+        try:
+            with websocket.expect_event("framereceived", waiter, timeout=timeout):
+                if press_start_button:
+                    _trigger_service_start(page, node_id)
+        except Exception:
```

Comment:
how do you know this timeout comes from the `with websocket.expect_event`? Note that `_trigger_service_start` also incorporates a timeout in the implementation!